HappyAnding/HappyAnding/Views/SignInViews/PrivacyPolicyView.swift

```diff
@@ -12,48 +12,78 @@ import WebKit
 
 class WebViewModel: ObservableObject {
     @Published var isLoading = false
+}
+
+struct PrivacyPolicyView: View {
+    
+    @ObservedObject var viewModel: WebViewModel
+    
+    @Binding var isTappedPrivacyButton: Bool
     
-    var url: String
+    let url: String
     
-    init(url: String) {
-        self.url = url 
+    var body: some View {
+        
+        VStack {
+            HStack {
```

Comment:
HStack의 default spacing 때문에 타이틀이 살짝 치우쳐 보입니다 !!
```suggestion
            HStack(spacing: 0) {
```